### ISSUE TITLE
Bouncing when flinging upwards to the top.

### ISSUE THREAD

**Issue by timimahoney** [johannilsson/android-pulltorefresh#16]
If you are somewhere in the list, then fling up to the top, the list bounces after a short delay.

Not sure why this is, and I haven't been able to fix it. It seems to happen right before the scroll state is set to idle. Any suggestions?

**Comment by timimahoney** [comment]
Fixed and submitted a pull request.

```
private boolean mBounceHack;

....in onScroll

else if (mCurrentScrollState == SCROLL_STATE_FLING && firstVisibleItem == 0 && mRefreshState != REFRESHING) {
    setSelection(1);
    mBounceHack = true;
} else if (mBounceHack) {
    setSelection(1);
}


... in onScrollStateChanged

if (mCurrentScrollState == SCROLL_STATE_IDLE) {
    mBounceHack = false;
}
```

Then, in public void onScrollStateChanged, when the scroll state changes to SCROLL_STATE_IDLE, we turn that mBounceHack flag back to false.

**Comment by timimahoney** [comment]
Actually. This didn't fix it compeltely. It introduces new bugs. You can't scroll back down until the internal fling has completed because this still thinks we need the hack.

**Comment by johannilsson** [comment]
Thank you for the report. This is strange, I can clearly see it now when you mentioned it but haven't seen it before though.

**Comment by timimahoney** [comment]
No problem, and thanks for the library! I think I fixed the problem with that latest pull request. The first pull request didn't work completely.

**Comment by johannilsson** [comment]
Fixed in #18, thank you.
